automatic/graphviz/legal/LICENSE.txt

```diff
@@ -1,53 +1,53 @@
-All Recipient's rights under this Agreement shall terminate if it fails to comply with any of the material terms or conditions of this Agreement and does not cure such failure in a reasonable period of time after becoming aware of such noncompliance. If all Recipient's rights under this Agreement terminate, Recipient agrees to cease use and distribution of the Program as soon as reasonably practicable. However, Recipient's obligations under this Agreement and any licenses granted by Recipient relating to the Program shall continue and survive. 
-Everyone is permitted to copy and distribute copies of this Agreement, but in order to avoid inconsistency the Agreement is copyrighted and may only be modified in the following manner. The Agreement Steward reserves the right to publish new versions (including revisions) of this Agreement from time to time. No one other than the Agreement Steward has the right to modify this Agreement. IBM is the initial Agreement Steward. IBM may assign the responsibility to serve as the Agreement Steward to a suitable separate entity. Each new version of the Agreement will be given a distinguishing version number. The Program (including Contributions) may always be distributed subject to the version of the Agreement under which it was received. In addition, after a new version of the Agreement is published, Contributor may elect to distribute the Program (including its Contributions) under the new version. Except as expressly stated in Sections 2(a) and 2(b) above, Recipient receives no rights or licenses to the intellectual property of any Contributor under this Agreement, whether expressly, by implication, estoppel or otherwise. All rights in the Program not expressly granted under this Agreement are reserved. 
-This Agreement is governed by the laws of the State of New York and the intellectual property laws of the United States of America. No party to this Agreement will bring a legal action under this Agreement more than one year after the cause of action arose. Each party waives its rights to a jury trial in any resulting litigation.
-
+AT&T has previously made versions of this software available under the AT&T Source Code Agreement, version 1.2D and earlier. If you received a copy of the software under that license agreement, you may continue to use and distribute the same version of the software subject to the terms and conditions of the license agreement under which the software was received. However, current versions of the software are now licensed on an open source basis only under The Common Public License. 

```

Comment:
Do you know what the source of this file is? I ask because it doesn't seem to match the LICENSE file in the top level of the Graphviz repo.

Reply:
I was just using the file from the coreteam-repo, but we can replace it with the [License](https://gitlab.com/graphviz/graphviz/-/blob/master/LICENSE) file possibly.
/cc @majkinetor @AdmiringWorm

Reply:
Sorry, what is the "coreteam-repo"?

Reply:
I just realized this is only even showing up in the diff because this PR switches all the line endings from unix to Windows. Having said that, automatic/legal/LICENSE.txt does appear to be an added file.

Reply:
@Smattr the coreteam-repo is this automated git  ;-)

Reply:
Meaning the present repository? I.e. the destination of this PR?

Reply:
> I just realized this is only even showing up in the diff because this PR switches all the line endings from unix to Windows

@RedBaron2 I can strongly recommend [core.autocrlf](https://git-scm.com/book/en/v2/Customizing-Git-Git-Configuration#_core_autocrlf) to avoid this.

Reply:
I will fix the line ending when I get back. A unfortunate error from when I first transferred the file from the repo.